server/src/pages/shotindex/view.js

```diff
@@ -248,6 +287,7 @@ class Card extends React.Component {
   }
 
   displayTitle(title) {
+    // todo l10n: this won't work for rtl languages. use CSS ellipsis instead?
```

Comment:
I recall overflow being wonky here, but the CSS has been refactored a few times so maybe it will work now.  In which case we might not need this at all, *or* we might just be able to lose the ellipsis (but keep the truncation) and let people figure it out.